package.json

```diff
@@ -0,0 +1,641 @@
+{
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you configure npm scripts to run your code  locally